pkg/collector/helm.go

```diff
@@ -20,22 +24,39 @@ func parseManifests(manifest string, defaultNamespace string) ([]map[string]inte
 			return nil, err
 		}
 
-		fixNamespace(&manifest, defaultNamespace)
+		fixed := fixNamespace(manifest, defaultNamespace, discoveryClient)
 
-		results = append(results, manifest)
+		results = append(results, fixed)
 	}
 
 	return results, nil
 }
 
-func fixNamespace(manifest *map[string]interface{}, defaultNamespace string) {
+func fixNamespace(manifest map[string]interface{}, defaultNamespace string, discoveryClient discovery.DiscoveryInterface) map[string]interface{} {
```

Comment:
I would prefer the pointer, can't we use the unstructured. can't we deference instead? for 

```
&unstructured.Unstructured{Object: *manifest}
```

Reply:
If its used else where ignore me